src/main/resources/changelog/changes/v3.3/create-reindex-entity-tables.xml

```diff
@@ -0,0 +1,282 @@
+        <constraints nullable="false"/>
+      </column>
+    </createTable>
+  </changeSet>
+
+  <changeSet id="MSEARCH-793@@create-holdings-table" author="Mukhiddin_Yusupov">
+    <preConditions>
+      <not>
+        <tableExists tableName="holdings"/>
+      </not>
+    </preConditions>
+
+    <comment>Create holdings table</comment>
+
+    <createTable tableName="holdings">
+      <column name="holding_id" type="VARCHAR(255)">
+        <constraints nullable="false" primaryKey="true" primaryKeyName="pk_holdings"/>
+      </column>
+      <column name="tenant_id" type="VARCHAR(255)">
+        <constraints nullable="false" primaryKey="true" primaryKeyName="pk_holdings"/>
+      </column>
+      <column name="instance_id" type="VARCHAR(255)">
+        <constraints nullable="false"/>
+      </column>
+      <column name="holding_json" type="jsonb">
+        <constraints nullable="false"/>
+      </column>
+    </createTable>
+  </changeSet>
+
+  <changeSet id="MSEARCH-793@@create-idx_holdings_instance_id" author="Mukhiddin_Yusupov">
+    <preConditions>
+      <not>
+        <indexExists indexName="idx_holdings_instance_id"/>
+      </not>
+    </preConditions>
+
+    <comment>Create idx_holdings_instance_id index</comment>
+
+    <createIndex tableName="holdings" indexName="idx_holdings_instance_id">
+      <column name="instance_id"/>
+    </createIndex>
+  </changeSet>
+
+  <changeSet id="MSEARCH-793@@create-items-table" author="Mukhiddin_Yusupov">
+    <preConditions>
+      <not>
+        <tableExists tableName="items"/>
+      </not>
+    </preConditions>
+
+    <comment>Create items table</comment>
+
+    <createTable tableName="items">
+      <column name="item_id" type="VARCHAR(255)">
+        <constraints nullable="false" primaryKey="true" primaryKeyName="pk_items"/>
+      </column>
+      <column name="tenant_id" type="VARCHAR(255)">
+        <constraints nullable="false" primaryKey="true" primaryKeyName="pk_items"/>
+      </column>
+      <column name="holding_id" type="VARCHAR(255)">
+        <constraints nullable="false"/>
+      </column>
+      <column name="item_json" type="jsonb">
+        <constraints nullable="false"/>
+      </column>
+    </createTable>
+  </changeSet>
+
+  <changeSet id="MSEARCH-793@@create-idx_items_holding_id" author="Mukhiddin_Yusupov">
+    <preConditions>
+      <not>
+        <indexExists indexName="idx_items_holding_id"/>
+      </not>
+    </preConditions>
+
+    <comment>Create idx_items_holding_id index</comment>
+
+    <createIndex tableName="items" indexName="idx_items_holding_id">
+      <column name="holding_id"/>
+    </createIndex>
+  </changeSet>
+
+  <changeSet id="MSEARCH-793@@create-classifications-table" author="Mukhiddin_Yusupov">
+    <preConditions>
+      <not>
+        <tableExists tableName="classifications"/>
+      </not>
+    </preConditions>
+
+    <comment>Create classifications table</comment>
+
+    <createTable tableName="classifications">
```

Comment:
Add additional id PK column. We should generate it similar to subjects and contributors.